### DISCUSSION TITLE
Terminal font and PS1 variable - Alacritty

### DISCUSSION THREAD

**Discussion by spcarluccio** [akinomyoga/ble.sh#201] [Q&A]
Hi, i just started using ble.sh on an arch linux + i3wm and alacritty as terminal. Why my font changed from the one choosen in my alacritty config (noto sans mono) and my PS1 prompt changed as well from .bashrc ?

**Comment by akinomyoga** [comment]
Thank you for the report. Hmm, `ble.sh` doesn't change either font or `PS1`. I suspect some other configurations affect them.
- Does it happen even with a plain Bash plus only your `PS1` and `ble.sh` (without any other configuration) in `.bashrc`?
- If it still happens, could you provide us with your `.bashrc`?
